.github/workflows/ci.yml

```diff
@@ -89,10 +89,8 @@ jobs:
       # check for missing docs
       - name: cargo check
         run: cargo check --all-features
-        env:
-          RUSTFLAGS: "--cfg docsrs"
```

Comment:
I feel like this line was important for something... There's a builder that has docs exposed but otherwise the type is unnameable, and I added this to catch that all methods were documented correctly. I think?

Reply:
Migrated this checking to the building document process. It could be detected as https://github.com/seanmonstar/warp/actions/runs/18195955801/job/51802257485#step:4:202.